pages/docs/tools/asyncapi-cli.md

```diff
@@ -0,0 +1,137 @@
+---
+title: AsyncAPI CLI
+weight: 1
+---
+
+## Install CLI Using NPM and Node
+To use the AsynAPI CLI tool, you must have Node.js version 10 or higher and NPM installed on your PC. To check if you already have both installed, run the following commands in your terminal:
+
+```sh
+# check if node is installed
+node -v
+# or
+node --version
+
+# check if NPM is installed
+npm -v
+# or
+npm --version
+```
+
+If you don’t have Node.js or NPM installed, [install both with this package manager](https://nodejs.org/en/download/package-manager/).
+
+After installing Node.js and NPM, run the following command to install the AsyncAPI CLI globally on your PC:
+```sh
+npm install -g @asyncapi/cli
+```
+
+## Install CLI on MacOS
+There are two ways to install the AsyncAPI CLI on your MacOS:
```

Comment:
```suggestion
There are two ways to install the AsyncAPI CLI on your macOS: using `brew` or `pkg`.
```

You should not leave a fragment dangling before you start a new header/sub-section. Always write complete sentences.